hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -155,9 +158,8 @@ public class SnapshotDiffManager implements AutoCloseable {
    */
   private final String sstBackupDirForSnapDiffJobs;
 
-  private boolean isNativeRocksToolsLoaded;
-
-  private ManagedSSTDumpTool sstDumpTool;
+  private final boolean isNativeRocksToolsLoaded;
```

Comment:
Nit: In hindsight we could have just gotten away with just an Optional.

Reply:
Yes, we can use `Optional` but it will require some changes in [addToObjectIdMap](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java#L741). I'll leave it as it is in this PR and will address in [PR-4617](https://github.com/apache/ozone/pull/4617) since I already did some refactoring in that.

Reply:
Changed it to `Optional` in https://github.com/apache/ozone/pull/4617/files#diff-46f82d5fbac5dc7aae1fc363811fb88e6bc6c89a2aefafddad90de2f1ff3cfbdR163